python/pyspark/pandas/tests/test_generic_functions.py

```diff
@@ -0,0 +1,124 @@
+#
```

Comment:
Oh, actually we should add this file into https://github.com/apache/spark/blob/master/dev/sparktestsupport/modules.py#L440

Reply:
let me make a quick followup

Reply:
https://github.com/apache/spark/pull/36271